ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -239,6 +239,107 @@ device_classes: !ZenPackSpec
                                 rrdmin: 0
                             missing:
                                 rrdmin: 0
+                graphs:
+                    Utilization:
+                        units: percent
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            size:
+                                dpName: volume_size
+                                lineType: DONTDRAW
+                            free:
+                                dpName: volume_free
+                                lineType: AREA
+                                rpn: "size,-,size,/,-100,*"
+                thresholds:
+                    unallocatable:
```

Comment:
It's not a good practice to use thresholds to create boolean status events like this when you have another option. In some cases like SNMP datasources you don't have any ability to do smarter processing on collected datapoints and have to resort using min/max thresholds like this to get status events. That's not the case with COMMAND datasources.

Instead of defining these thresholds, the parser should be creating proper events. Otherwise you get ugly events saying something like "threshold of unallocatable not met: current value 0.0". That doesn't seem like the right message to give the user.